### DISCUSSION TITLE
Refresh Filament Table component

### DISCUSSION THREAD

**Discussion by Pplan47** [filamentphp/filament#13179] [Help] [answered]
### Package

Table builder

### Package Version

v3.0.0

### How can we help you?

I have a Livewire component page with some buttons and a Filament Table component.
How can I refresh the Table from one of the buttons on the view?
Can anyone put me in the right direction to solve the problem?

Thanks in advance!

**Comment by leandrocfe** [comment, marked as answer]
Try this: https://filamentphp.com/docs/3.x/tables/advanced#resetting-the-table
If it isn't work, share the code you are using

**Reply by Pplan47** [reply]
Thanks a lot!

$this->resetTable();  works

**Comment by CoringaWc** [comment]
The resetTable() method is not working in the relation manager of the resource when changing the disabled state of a ToggleColumn.
I tried calling resetTable() inside both the updateStateUsing and afterStateUpdated callbacks, but the ToggleColumn does not update its state.
I also tried triggering a Livewire event to force the table to refresh, but that didn’t work either.
